File: src/TmrOverlay.App/Overlays/PitService/PitServiceOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.PitService;

internal static class PitServiceOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "pit-service",
        DisplayName: "Pit Service",
        DefaultWidth: 530,
        DefaultHeight: 743,
        FadeWhenLiveTelemetryUnavailable: true,
        ContextRequirement: OverlayContextRequirement.LocalPlayerInCarOrPit);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/PitService/PitServiceOverlayDefinition.swift
import AppKit

enum PitServiceOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "pit-service",
        displayName: "Pit Service",
        defaultSize: NSSize(width: 420, height: 430),
        fadeWhenLiveTelemetryUnavailable: true
    )
}
